multi-agent/lib/mas/core/auth/schemes/oauth2/adapter.py

```
@@ -0,0 +1,223 @@
+        config: Dict[str, Any],
+        code: str,
+        code_verifier: Optional[str],
+        redirect_uri: str,
+    ) -> TokenSet:
+        cfg = OAuth2Config.from_dict(config)
+
+        try:
+            async with AsyncOAuth2Client(
+                client_id=cfg.client_id,
+                client_secret=cfg.client_secret,
+                token_endpoint_auth_method="client_secret_post",
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token_endpoint_auth_method"

(S106)